packages/grafana-runtime/src/types/analytics.ts

```diff
@@ -1,12 +1,19 @@
 import { EchoEvent, EchoEventType } from '../services/EchoSrv';
 
+/**
+ * Describes the basic dashboard information that can be passed as the meta
+ * analytics payload.
+ */
 export interface DashboardInfo {
```

Comment:
note: I don't really see the reason why we want to export the "base" interfaces outside the runtime package.